### PR TITLE
Update text for generated readme file

### PR DESCRIPTION
**Pull request by davydovanton**
Reference to new discussion here: https://github.com/hanami/hanami/issues/759

Thanks @cllns and @TiteiKo

### CODE REVIEW

File: lib/hanami/generators/application/container/README.md.tt [hunk 1]
```diff
@@ -2,16 +2,30 @@
 
 Welcome to your new hanami project!
```
Comment (jodosha) [inline lib/hanami/generators/application/container/README.md.tt]: s/hanami/Hanami/

File: lib/hanami/generators/application/container/README.md.tt [hunk 2]
```diff
@@ -2,16 +2,30 @@
...
 
-TODO: Delete this and the text above, and describe your project
+## Useful informations
```
Comment (jodosha) [inline lib/hanami/generators/application/container/README.md.tt]: Maybe "Setup"?

File: lib/hanami/generators/application/container/README.md.tt [hunk 3]
```diff
@@ -2,16 +2,30 @@
...
 
-## Installation
+How to run tests:
+```
+% bundle exec rake
+```
 
-TODO: Delete this text and describe installation process
+How to run a development console:
```
Comment (jodosha) [inline lib/hanami/generators/application/container/README.md.tt]: s/a/the

File: lib/hanami/generators/application/container/README.md.tt [hunk 4]
```diff
@@ -2,16 +2,30 @@
...
+```
+% bundle exec hanami console
+```
 
-## Usage
+How to run a development server:
```
Comment (jodosha) [inline lib/hanami/generators/application/container/README.md.tt]: s/a/the

File: lib/hanami/generators/application/container/README.md.tt [hunk 5]
```diff
@@ -2,16 +2,30 @@
...
+% bundle exec hanami db create
+% bundle exec hanami db migrate
 
-## License
+% HANAMI_ENV=test bundle exec hanami db create
+% HANAMI_ENV=test bundle exec hanami db migrate
+```
 
-The project is available as open source under the terms of the [MIT License](http://opensource.org/licenses/MIT).
+Explore the other [guides](http://hanamirb.org/guides/), the Hanami [API documentation](http://hanamirb.org/docs/1.0.0/) and read [the source code](http://github.com/hanami). Enjoy!
```
Comment (jodosha) [inline lib/hanami/generators/application/container/README.md.tt]: What do you think about this?

```
Explore Hanami [guides](http://hanamirb.org/guides/), [API docs](http://hanamirb.org/docs/<%= Hanami::VERSION %>/), or jump in [chat](http://chat.hanamirb.org) for help. Enjoy! 🌸
```

Comment (davydovanton) [inline lib/hanami/generators/application/container/README.md.tt]: great! 👏

**Comment by davydovanton** [comment]
@jodosha I fixed you issues, thanks!

@AlfonsoUceda @TiteiKo thanks for review 💚

File: lib/hanami/generators/application/container/README.md.tt [hunk 6]
```diff
@@ -1,17 +1,31 @@
...
 
-Welcome to your new hanami project!
+Welcome to your new Hanami project!
 
-TODO: Delete this and the text above, and describe your project
+## Setup
 
-## Installation
+How to run tests:
+```
```
Comment (jodosha) [inline lib/hanami/generators/application/container/README.md.tt]: @davydovanton Please leave a blank line between the paragraph and the beginning of the three backticks. Some Markdown renderer are getting confused by this. For instance iA Writer on Mac.

**Comment by davydovanton** [comment]
thanks for the review! 💚

**Pull request merged**